src/main/java/com/redhat/cloud/notifications/NotificationsGwApp.java

```diff
@@ -21,30 +21,44 @@
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.InputStreamReader;
+import java.util.Optional;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 import org.eclipse.microprofile.config.inject.ConfigProperty;
 import org.jboss.logging.Logger;
 
+import javax.enterprise.context.ApplicationScoped;
 import javax.enterprise.event.Observes;
 
 /**
  * @author hrupp
  */
+@ApplicationScoped
 public class NotificationsGwApp {
 
+    private static final String NOTIFICATIONS_URL_KEY = "notifications-backend/mp-rest/url";
+
     public static final String FILTER_REGEX = ".*(/health(/\\w+)?|/metrics) HTTP/[0-9].[0-9]\" 200.*\\n?";
     private static final Pattern pattern = Pattern.compile(FILTER_REGEX);
 
     @ConfigProperty(name = "quarkus.http.access-log.category")
     private String loggerName;
 
+    @ConfigProperty(name = "clowder.endpoints.notifications-backend-service")
+    Optional<String> notificationsBackendClowderEndpoint;
+
     private static final Logger LOG = Logger.getLogger(NotificationsGwApp.class);
 
     void init(@Observes StartupEvent ev) {
         initAccessLogFilter();
 
         LOG.info(readGitProperties());
+
+        if (notificationsBackendClowderEndpoint.isPresent()) {
+            String notificationsUrl = "http://" + notificationsBackendClowderEndpoint.get();
+            LOG.infof("Overriding the notifications-backend URL with the config value from Clowder: %s", notificationsUrl);
+            System.setProperty(NOTIFICATIONS_URL_KEY, notificationsUrl);
+        }
```

Comment:
`clowder-quarkus-config-source` got updated and made things simpler, this (and the `ConfigProperty` above) is no longer required.

See https://github.com/RedHatInsights/notifications-backend/pull/812 (application.properties) for the new way of using Clowder endpoints.